### PR TITLE
Add search option to GQL schema

### LINKED ISSUE #3131: Add search option to GQL schema.
In [barista/issues/287](https://github.com/eventespresso/barista/issues/287), we need to be able to search for entities in a searchable input. Our GQL Schema does not have the search feature yet.
Looking into our PHP Model system, it looks like our model system doesn't have anything out of the box either.

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR adds search support to GQL schema for EE entities.

Closes #3131

### CODE REVIEW

File: docs/GraphQL-API/query/attendee.md [hunk 1]
```diff
@@ -6,31 +6,31 @@ Attendee object has connections with `RootQuery`.
...
-					name
-				}
-				state {
-					name
-				}
-			}
-		}
-	}
+    espressoAttendees(where: $where) {
+        edges {
```
Comment (tn3rb) [inline docs/GraphQL-API/query/attendee.md:10]: can you plz fix your formatting and put it back to tabs instead of spaces for indents

**Review by tn3rb — CHANGES_REQUESTED** [review]
code looks great but your indents have changed to spaces when they should be tabs

File: docs/GraphQL-API/query/attendee.md (hunk 1 above)
Comment (manzoorwanijk) [reply docs/GraphQL-API/query/attendee.md]: It was prettier messing up. Had to update `.editorconfig` to fix it.

**Review by tn3rb — APPROVED** [review]
ya looks great

**Review by nicolad — APPROVED** [review]

**Pull request merged**